ray_on_golem/server/cluster/cluster.py

```diff
@@ -17,15 +20,15 @@
     ProviderParametersData,
     Tags,
 )
-from ray_on_golem.server.services.golem import (
+from ray_on_golem.server.services import (
     DriverListAllocationPaymentManager,
     GolemService,
     ManagerStack,
 )
 
 IsHeadNode = bool
```

Comment:
I still believe it makes absolutely zero sense to assign `bool` to another, custom type name...

Reply:
This is changed in #242. I'll keep the current state here to avoid merge conflicts.